Description: Add dependency on slf4j as this is required to support
 use of JCaptcha 2.0 (upstream uses 1.x).
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: jenkins/core/pom.xml
===================================================================
--- jenkins.orig/core/pom.xml	2012-01-31 10:35:35.048316342 +0000
+++ jenkins/core/pom.xml	2012-01-31 10:35:36.104316316 +0000
@@ -435,6 +435,12 @@
       <artifactId>memory-monitor</artifactId>
       <version>1.5</version>
     </dependency>
+    <!-- JCaptcha 2.0 uses slf4j which needs an implementation class -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-jdk14</artifactId>
+      <version>1.5.6</version>
+    </dependency>
     <dependency><!-- StAX implementation. See HUDSON-2547. -->
       <groupId>org.codehaus.woodstox</groupId>
       <artifactId>wstx-asl</artifactId>
